packages/core/src/components/overflow-list/overflowList.tsx

```diff
@@ -166,10 +201,14 @@ export class OverflowList<T> extends React.PureComponent<IOverflowListProps<T>,
             return;
         }
         if (growing) {
-            this.setState({
+            this.setState(state => ({
+                direction: OverflowDirection.GROW,
+                // store last overflow if this is the beginning of a resize (for check in `update()`).
```

Comment:
can you call this `componentDidUpdate` or remove the backticks and parens so it's clearer what you mean?